client-java/instrumentation/src/main/java/org/evomaster/client/java/instrumentation/object/ClassToSchema.java

```diff
@@ -72,17 +79,26 @@ public static void registerSchemaIfNeeded(Class<?> valueType) {
             return;
         }
 
-        String name = valueType.getName();
-        if (!UnitsInfoRecorder.isDtoSchemaRegister(name)){
-            List<Class<?>> embedded = new ArrayList<>();
-            String schema = ClassToSchema.getOrDeriveSchema(valueType, embedded);
-            UnitsInfoRecorder.registerNewParsedDto(name, schema);
-            ExecutionTracer.addParsedDtoName(name);
-            if (!embedded.isEmpty()){
-                embedded.forEach(ClassToSchema::registerSchemaIfNeeded);
-            }
+        try{
+            String name = valueType.getName();
+            if (!UnitsInfoRecorder.isDtoSchemaRegister(name)){
+                List<Class<?>> embedded = new ArrayList<>();
+                String schema = ClassToSchema.getOrDeriveSchema(valueType, embedded);
+                UnitsInfoRecorder.registerNewParsedDto(name, schema);
+                ExecutionTracer.addParsedDtoName(name);
+                if (!embedded.isEmpty()){
+                    embedded.forEach(ClassToSchema::registerSchemaIfNeeded);
+                }
 
+            }
+        }catch (Exception e){
+            SimpleLogger.warn("Fail to get schema for Class:"+valueType.getName(), e);
+            /*
+                fail with tests
+             */
+            assert(false);
```

Comment:
`assert(false)` is fine in core, but should avoided in driver, as user might start driver with `-ea` flag